content/en/network_monitoring/performance/setup.md

```diff
@@ -23,7 +23,7 @@ Datadog Network Performance Monitoring (NPM) gives you visibility into your netw
 - Identify outages of cloud provider regions and third-party tools.
 - Troubleshoot faulty service discovery with DNS server metrics.
 
-Network performance monitoring requires [Datadog Agent v6.14+][1].
+Network performance monitoring requires [Datadog Agent v6.14+][1] as well as enabling the [Live Process](https://docs.datadoghq.com/infrastructure/process/) collection feature as described in this [installation](https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows#installation) section.
```

Comment:
I don't think we should include this. This was only true for a few versions past 6.14, and live processes is a billed feature.

Reply:
Understood, I made changes and resubmitted.